sync/sync-impl/src/main/java/com/duckduckgo/sync/impl/ui/qrcode/SyncBarcodeView.kt

```diff
@@ -110,6 +131,16 @@ constructor(
         }
     }
 
+    private fun initQRScanner() {
+        if (syncFeature.qrCodeScannerHints().isEnabled()) {
+            if (appBuildConfig.isInternalBuild()) {
+                beepManager = BeepManager(getActivity())
```

Comment:
want to switch the order of these two `if` statements? 

`appBuildConfig.isInternalBuild()` is _cheaper_ than `syncFeature.qrCodeScannerHints().isEnabled()` so would be good to check that first

also, can you get `syncFeature.qrCodeScannerHints().isEnabled()` off of the main thread?